test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-ui-state.json

```diff
@@ -247,6 +247,7 @@
       "swapsQuoteRefreshTime": 60000,
       "swapsQuotePrefetchingRefreshTime": 60000,
       "swapsStxBatchStatusRefreshTime": 10000,
+      "swapsStxStatusDeadline": "number",
```

Comment:
Why is this a "number" string and not some exact number like other fallback values?

Reply:
The value wasn't being set up correctly in the test, but it is now 👍